src/util/luxon-utils.ts

```
@@ -9,20 +9,25 @@
- * @param value - the {@link LuxonValue} to check.
- * @param message - the message to use in the error.
- * @throws error if value is invalid.
+ * Asserts that the provided Luxon value is valid.
+ *
+ * This function checks whether a Luxon date, duration, or interval is valid. If the value is invalid,
+ * it throws an error with a message combining a custom header (or default constructor name) with the
+ * value's invalid reason and, if available, its invalid explanation.
+ *
+ * @param value - The Luxon object to validate.
+ * @param message - Optional custom header for the error message.
+ *
```

ESLint

[error] Expected no lines between tags

(jsdoc/tag-lines)